variables.tf

```diff
@@ -29,6 +29,17 @@ variable "members" {
   }
 }
 
+variable "computed_members_map" {
+  type        = map(string)
+  description = "(Optional) A map of members to replace in 'var.members' to handle terraform computed values. Will be ignored when policy bindings are used."
+  default     = {}
+
+  validation {
+    condition     = alltrue([for k, v in var.computed_members_map : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain):)", v))])
```

Comment:
please update the condition according to the [resource's documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/artifact_registry_repository_iam#member/members). 
aka. copy paste as i did before, what is wrong that i know now thanks to @mariux :)
is this a minus beer?

Reply:
no worries, actually there is a beer counter from @mariux for every time i made this mistake

Reply:
hahahaha add me up for 1 beer then